Solutions/PaloAltoPrismaCloud/Hunting Queries/PaloAltoPrismaCloudUpdatedResources.yaml

```diff
@@ -0,0 +1,23 @@
+id: d0a145c5-546d-48df-a5d7-8866f3bbe24f
+name: Palo Alto Prisma Cloud - Updated resources
+description: |
+  'Query searches recently updated resources.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: PaloAltoPrismaCloud
+    dataTypes:
+      - PaloAltoPrismaCloud
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1133
+query: |
+  PaloAltoPrismaCloud
+  | where TimeGenerated > ago(24h)
+  | Reason =~ 'RESOURCE_UPDATED'

```

Comment:
@vu-socprime , Please correct the query by adding  where before Reason to fix.
| **where** Reason =~ 'RESOURCE_UPDATED'